pact/Pact/Core/Gas.hs

```diff
@@ -166,8 +149,10 @@ data GasArgs
   | GSearch !SearchType
   -- ^ Gas costs for searches
   | GPoseidonHashHackAChain !Int
-  -- ^ poseidon-hash-hack-a-chain costs
+  -- ^ poseidon-hash-hack-a-chain costs.
   | GModuleMemory !Word64
+  | GCountBytes
```

Comment:
Missing the `NBytes` argument?

Reply:
This is just the cost of calling `countBytes`, which happens during (incremental) `SizeOf` computations. That cost doesn't depend on how many bytes get counted.